### PR TITLE
Update LocalSettings.php with MFE configs

### PR DESCRIPTION
**Pull request by ugochimobi**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -2935,8 +2936,9 @@
 	],
 	'wgMFNoindexPages' => [
 		'wmgUseMobileFrontend' => false,
-		'gratisdatawiki' => true,
-		'gratispaideiawiki' => true,
+		'default' => true,
```
Comment (Universal-Omega) [inline LocalSettings.php:2939]: No. This should not be done..the default is already false, by the wmgUseMobileFrontend part of this. Which makes it only set config if MobileFrontend is enabled.

**Review by Universal-Omega — CHANGES_REQUESTED** [review]
+conflict. Not extensively reviewed.

**Comment by ugochimobi** [comment]
@Universal_Omega, the 'default' should be removed now?

File: LocalSettings.php (hunk 1 above)
Comment (ugochimobi) [reply LocalSettings.php:2939]: The default is supposed to be true according to extension.json https://github.com/wikimedia/mediawiki-extensions-MobileFrontend/blob/master/extension.json#L870
So what I want is false for my wikis.

**Comment by ugochimobi** [comment]
@Universal-Omega I don't know how to resolve the other conflict

File: LocalSettings.php (hunk 1 above)
Comment (RhinosF1) [reply LocalSettings.php]: This does the opposite

Comment (ugochimobi) [reply LocalSettings.php]: @RhinosF1 how do you mean?

**Comment by ugochimobi** [comment]
Well, think I'd have to open separate PRs for each config as I actually don't know what causes the conflict.

**Pull request closed without merging**